digitalpalireader/content/js/translate.js

```diff
@@ -971,6 +974,27 @@ function changeAlt(e,i) {
   translateText(G_altChoices);
 }
 
+function insertWordByWord() {
```

Comment:
Any reason why this new code is required in web but wasn't required in xul?

Reply:
In xul, this js code was inside the `translate.htm`; however that goes against our rule to separate model and view.